app/javascript/application.js

```diff
@@ -0,0 +1,6 @@
+// Configure your import map in config/importmap.rb. Read more: https://github.com/rails/importmap-rails
```

Comment:
- [ ] All the previous requirements for `posts` such as creating the respective route, creating method and view, also apply for the creation of the comments